lib/rage/router/dsl.rb

```diff
@@ -129,6 +129,54 @@ def match(path, to:, constraints: {}, defaults: nil, via: :all)
       end
     end
 
+    def resources(name, constraints: nil, defaults: nil, only: nil, except: nil)
+      @path_prefixes << name.to_s
+
+      yield if block_given?
+
+      @path_prefixes.pop
+
+      default_resource_actions = %i[index show create update destroy]
+      raise ArgumentError, "only one of 'only' and 'except' options can be specified" if only && except
+
+      routes = []
+      if only
+        routes = only
+        routes.each do |route|
+          raise ArgumentError, "Bad resource route: #{route} for only option" unless default_resource_actions.include?(route)
+        end
+      elsif except
+        routes = default_resource_actions - except
+      else
+        routes = default_resource_actions
+      end
+
+      routes.each do |route|
+        case route
+        when :index
+          __on("GET", "#{name}", "#{name}#index", constraints, defaults)
+        when :show
+          __on("GET", "#{name}/:id", "#{name}#show", constraints, defaults)
+        when :create
+          __on("POST", "#{name}", "#{name}#create", constraints, defaults)
+        when :update
+          __on("PUT", "#{name}/:id", "#{name}#update", constraints, defaults)
+        when :destroy
+          __on("DELETE", "#{name}/:id", "#{name}#destroy", constraints, defaults)
+        end
+      end
+    end
+
+    def namespace(path, &block)
```

Comment:
It should also support the `path` and `module` options.

https://api.rubyonrails.org/classes/ActionDispatch/Routing/Mapper/Scoping.html#method-i-namespace